### DISCUSSION TITLE
ArcoLinux touch

### DISCUSSION THREAD

**Discussion by arcolinuxz** [bakkeby/dusk#2] [General]
Great work.

https://github.com/arcolinux/arcolinux-dusk
![ArcoLinux_2021-10-15_16-12-39](https://user-images.githubusercontent.com/36679406/137528513-402caa3e-8c30-4635-86a1-74b44ec448f7.png)
![ArcoLinux_2021-10-15_16-12-15](https://user-images.githubusercontent.com/36679406/137528523-e17a5070-dee0-4452-8cdc-c33d47933e51.png)

**Comment by rayvermey** [comment]
Nice done Erik!

**Comment by bakkeby** [comment]
Thank you, that is quite an honour.

**Comment by arcolinuxz** [comment]
Part of the ArcoLinux eco-system now :-)
ArcoLinux Tweak Tool
![att-dusk](https://user-images.githubusercontent.com/36679406/137629643-3900af13-4038-4fe0-8988-48caad17f5c9.jpg)

ArcoLinux Calamares
![dusk-calamares](https://user-images.githubusercontent.com/36679406/137629705-ef3ddc13-ace6-4502-9814-63c7c080da31.png)

ArcoLinuxB Xtended
![arcolinuxb-xtended](https://user-images.githubusercontent.com/36679406/137629711-1082f0ee-c6e1-4ad4-a91f-a517437ef841.png)

**Comment by OpAmlee** [comment]
this is really nice. using as the daily from now on. love the centered quick terminal windows on mod+w. tons of cool features to explore. thank you for your work on this!
